### PR TITLE
Feature/mkk 17 create route employee

### PR DESCRIPTION
**Pull request by kendrickcarrolldc**
Created table for employee app with mock information and ui-view.

### CODE REVIEW

**Comment by anishrijal** [comment]
its ok but u should work on the nav bar, coz when you are logged in u still have login button instead of logout

File: client/app/main/main.js [hunk 1]
```diff
@@ -5,5 +5,15 @@ angular.module('maerkApp')
     $stateProvider.state('main', {
       url: '/',
       template: '<main class="flex layout column"></main>'
```
Comment (lchapoy) [inline client/app/main/main.js]: Change this route to be abstact.
 If main route is not abstract you will have an error when reloading

**Review by lchapoy — CHANGES_REQUESTED** [review]

File: client/app/main/main.js [hunk 2]
```diff
@@ -3,7 +3,19 @@
...
   .config(function($stateProvider) {
     $stateProvider.state('main', {
-      url: '/',
-      template: '<main class="flex layout column"></main>'
-    });
+      url: '/dashboard',
+      template: '<main class="flex layout column"></main>',
+			authenticate: true,
+			abstract:truegi
+    })
```
Comment (lchapoy) [inline client/app/main/main.js:10]: abstract: truegi??? what is that

File: client/app/account/account.js [hunk 3]
```diff
@@ -30,8 +30,15 @@ angular.module('maerkApp')
         controller: 'SettingsController',
         controllerAs: 'vm',
         authenticate: true
-      });
+      })
```
Comment (lchapoy) [inline client/app/account/account.js:33]: Erase thing that you are not using

**Review by lchapoy — APPROVED** [review]

**Pull request merged**